packages/reakit/src/styled.ts

```diff
@@ -20,5 +10,6 @@ export {
   withTheme,
   ServerStyleSheet,
   StyleSheetManager
-};
+} from "styled-components";
```

Comment:
@ewgenius Just a question (I'm investigating some bugs on build here): was there any specific reason for this change?